src/interfaces/interface.ts

```diff
@@ -0,0 +1,18 @@
+export interface ICompetency {
```

Comment:
I would prefer it if each interface was its own file

Reply:
or even just a meaningful filename, if all the interfaces fall into a similar group